### DISCUSSION TITLE
自动肉鸽疑似将其他藏品误认为是指路鳞

### DISCUSSION THREAD

**Discussion by gerald-liu** [MaaAssistantArknights/MaaAssistantArknights#5087] [Q&A]
可以看到，Log显示已购买目标商品，按理说是指路鳞：
![1](https://github.com/MaaAssistantArknights/MaaAssistantArknights/assets/36258909/f5dfa08f-cd40-4ebe-a609-3861de6dd4a1)

但是结束任务之后查看藏品列表，并没有点亮指路鳞：
![2](https://github.com/MaaAssistantArknights/MaaAssistantArknights/assets/36258909/dd619278-bb21-4fec-a3e3-cec67c7d4459)

该时段对应的asst.log如下：
[asst.log.20230607.txt](https://github.com/MaaAssistantArknights/MaaAssistantArknights/files/11673618/asst.log.20230607.txt)

**Comment by ABA2396** [comment]
也没发log啊

**Reply by gerald-liu** [reply]
加上去了，你看看

**Comment by MistEO** [comment]
是指路鳞，源石锭不够没买上

**Reply by gerald-liu** [reply]
明白了，那GUI的提示信息可以考虑改成“已发现目标商品”之类的，买上了再出一条已购买
